### PR TITLE
Purchases: Fix credit card information misaligned in Manage Purchase on mobile (Fixes #200)

### LINKED ISSUE #200: Purchases: credit card information misaligned in Manage Purchase, mobile
The credit card information in Manage Purchase is aligned to the left in smaller devices. All the other details are aligned to the right.

Screenshot from Firefox 40.0.3 for Linux, 320px wide:

screen shot 2015-11-19 at 17 42 37

### PR DESCRIPTION
**Pull request by umurkontaci**
This fixes a CSS issue where the credit card information is aligned to left on small screens.

/cc: @fabianapsimoes

### CODE REVIEW

**Comment by fabianapsimoes** [comment]
Looks good :+1: 

screenshot-calypso localhost 3000 2015-11-24 19-50-02

**Comment by mikeshelton1503** [comment]
The CC info is fixed, however I noticed that "included with plan" is also broken:

https://cloudup.com/cBfUM1T_WHR
image

File: client/me/purchases/manage-purchase/index.jsx [hunk 1]
```diff
@@ -288,7 +288,7 @@ const ManagePurchase = React.createClass( {
 			}
 
 			return (
-				<span className="manage-purchase__payment-info">
+				<span className="manage-purchase__detail manage-purchase__payment-info">
```
Comment (drewblaisdell) [inline client/me/purchases/manage-purchase/index.jsx]: I think this should only have one class, `manage-purchase__detail`. We can style the child by using the `PaymentLogo` classname.

**Comment by drewblaisdell** [comment]
Sorry, I didn't realize there was an issue for this and fixed it in #670. :sweat:

**Comment by umurkontaci** [comment]
Everything about this PR is fixed in #670 and this is void now.

**Pull request closed without merging**